projects/js-packages/publicize-components/src/components/schedule-button/index.tsx

```diff
@@ -22,6 +22,10 @@ interface ScheduleButtonProps extends ScheduleButtonBaseProps {
 	isDisabled?: boolean;
 }
 
+const isInvalidDate = ( checkDate: Date ) => {
+	return checkDate.getTime() < Date.now();
```

Comment:
I think this has to be checked against the site's timezone, not the browser timezone.

Reply:
Or it's better to use the `isInTheFuture` function from `@wordpress/date`

Reply:
Yes, I've been scratching my head with some of the timezone stuff. I've updated it to only use the `@wordpress/date` functions, so that handles the timezone adjustments.